Resources/Prototypes/_Backmen/Entities/Objects/Throwable/throwing_stars.yml

```
@@ -1,13 +1 @@
-  id: ThrowingStarChangeling
-  name: bone shard
-  components:
-  - type: Sprite
-    sprite: Backmen/Changeling/bone_shard.rsi
-    layers:
-    - state: icon
-      map: ["base"]
-  - type: TimedDespawn
-    lifetime: 30
-
+﻿
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)